study.md

```diff
@@ -43,21 +43,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- View is how many times anyone visits a website.  view state controls how many users can visit a website in a given period of time. -->
```

Comment:
The view is what a user sees and the view state is the state of the application at any given time.